app/views/organizations/adoptable_pets/index.html.erb

```diff
@@ -26,22 +28,24 @@
                 <%= f.label :sex_eq, "Sex" %>
                 <%= f.select :sex_eq, ["Male", "Female"], {include_blank: 'All'}, class: "form-select" %>
               </div>
-              <div class="form-group">
-                <%= f.label :species_eq, "Species" %>
-                <%= f.select :species_eq, Pet.species, {include_blank: 'All'}, class: "form-select" %>
-              </div>              
               <div class="form-group">
                 <%= f.label :breed_eq, "Breed" %>
-                <%= f.select :breed_eq, get_animals, {include_blank: 'All'}, class: "form-select" %>
+                <%= f.select :breed_eq,
```

Comment:
FYI the filter CSS is different between species. It would be good to make the same.
![filters](https://github.com/user-attachments/assets/a53dfcd7-1f2e-41c0-a6cd-62176182139e)

Reply:
I agree it seems abrupt switching between dogs/cats. I think(?) the solution would be to hard code the width of the longest named breed. The problem I see is that the max width of widest dog breed is different than the max width of the cat breeds. This would all be based off of our seed data if I were to set this.. What if there is a longer breed added to the DB than our seeds or a different species has an even longer breed than we can account for?

Going back to your point on most user's having a specific species in mind, I do not believe a user would be switching between different species and notice the change in spacing. I am hesitant to hardcore this but maybe I missed something?